_data/navigation.yml

```diff
@@ -1,8 +1,8 @@
 main:
   - title: Home
     url: /
-  - title: Blog
-    url: /blog/
+#  - title: Blog
```

Comment:
## Deactivated blog
I know that Shahim is fond of the idea of having a blog, and I see the logic. However, until we get an entry there, I felt it best to deactivate.